teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+class Teacher < Person
+  def initialize(specialization = 'Unknown')
+    super(age, name)
+    @specialization = specialization
```

Comment:
- [ ] While initializing the Teacher class, the constructor should extends that of the person class. the name, age and the parent_permission should be passed when creating a teacher. Kindly fix that.